src/app/component-library/edit-asset-form/edit-asset-form/edit-asset-form.component.ts

```diff
@@ -18,5 +19,10 @@ export class EditAssetFormComponent {
     public form: EditAssetForm,
     public validationMessages: ValidationMessages,
     public expressionFormHandler: ExpressionFormHandler,
+    public activeFeatureSet: ActiveFeatureSet,
   ) {}
+
+  multipleDataSourceOptionsAvailable(): boolean {
+    return this.form.mode !== 'CREATE' && !this.activeFeatureSet.hasMdsFields();
+  }
```

Comment:
This logic is not correct

Now MDS Edit looks like this

<img width="2560" alt="image" src="https://github.com/user-attachments/assets/038a71a7-70de-43af-a8eb-ad4d8ac1d5fe">

And MDS create does not have the options

<img width="2105" alt="image" src="https://github.com/user-attachments/assets/c6d13445-20b4-4c38-a476-45c814f5eb1a">

it should be this probably?

```this.form.mode === 'EDIT' || this.activeFeatureSet.hasMdsFields();```

Reply:
it works, thanks